ecommerce_integrations/shopify/order.py

```diff
@@ -157,7 +158,7 @@ def get_order_taxes(shopify_order, setting):
 			{
 				"charge_type": _("On Net Total"),
 				"account_head": get_tax_account_head(tax),
-				"description": f"{get_tax_account_description(tax) or tax.get('title')} - {tax.get('rate') * 100.0}%",
+				"description": f"{get_tax_account_description(tax) or tax.get('title')} - {round(Decimal(tax.get('rate') * 100.0), 2)}%",
```

Comment:
```suggestion
				"description": f"{get_tax_account_description(tax) or tax.get('title')} - {tax.get('rate') * 100.0:.2f}%",
```


No need to use rounding for this. It's just for sake of showing the description and isn't actually used for any calculation here. You can just use the f-string format specifier here.